Microsoft.Toolkit.Uwp.SampleApp/SamplePages/AnimationSet/AnimationSetPage.xaml

```diff
@@ -0,0 +1,45 @@
+﻿<Page
```

Comment:
This file needs running through xaml styler (https://github.com/Microsoft/UWPCommunityToolkit/blob/dev/contributing.md#quality-insurance-for-pull-requests-for-xaml-controls)